src/bot.ts

```diff
@@ -37,9 +37,12 @@ export default class Bot {
       logger.debug('ignored self tweet')
       return
     }
+    if (tweet.retweeted_status !== undefined) {
```

Comment:
In fact, I tried to avoid comparing vars directly with `undefined` since it can be modified at certain environments.
http://2ality.com/2013/04/check-undefined.html

But it should be no problem in Node.js environment 👍